src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", "Arial", sans-serif;
+  font-weight: 500;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo_section {
+  display: flex;
+  flex-grow: 1;
```

Comment:
strange solution, I think more logical to use `justify-content: space-between` to parent instead